### PR TITLE
chore: Update service versions to Mender 3.5.1 release.

### PR DESCRIPTION
**Pull request by kacf**
Changelog: Title
Ticket: None

### CODE REVIEW

File: mender/templates/gui-deploy.yaml [hunk 1]
```diff
@@ -62,7 +62,7 @@ spec:
 
         env:
         - name: INTEGRATION_VERSION
-          value: "3.5.0"
+          value: "3.5.1"
         - name: MENDER_VERSION
           value: "3.5.0"
```
Comment (kacf) [inline mender/templates/gui-deploy.yaml:67]: @mzedel: Can you confirm that this is correct? The Mender client version is still 3.5.0, IOW the Debian package. But the QEMU image is still part of the release, and is therefore tagged as `mender-3.5.1`. Will it treat it correctly?

**Comment by kacf** [comment]
Whoever reviews this, can I ask that you also do a quick sanity check that it runs?

File: mender/templates/gui-deploy.yaml (hunk 1 above)
Comment (mzedel) [reply mender/templates/gui-deploy.yaml:67]: Looks correct to me!
The client version should only be relevant for the version information in the lower left corner, the device connection snippets all rely on what the install script installs (afaik the latest stable release) or use the untagged docker image (so `latest`)...

Comment (kacf) [reply mender/templates/gui-deploy.yaml:67]: Ok, that should all work! Thanks!

**Review by tranchitella — APPROVED** [review]

**Pull request merged**